.env-example

```diff
@@ -0,0 +1,3 @@
+METACULUS_USERNAME=oughttest
```

Comment:
The tests will only work if you use the oughttest account (because they use private questions from that account)

So, we want to mandate in the example .env file that you should use that account

Obviously this is a little weird, but I figured it was a pretty simple way to do this for now that's still very extensible

I think I'll go ahead and try to publish the test questions on Metaculus so we can do away with this soon -- @stuhlmueller , sound good?